uni/lib/view/common_widgets/pages_layouts/general/widgets/bottom_navigation_bar.dart

```diff
@@ -42,6 +38,16 @@ class AppBottomNavbar extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     final currentIndex = _getCurrentIndex(context);
+    final navbarItems = <BottomNavigationBarItem>[];
```

Comment:
Any reason for dropping the funcional style? I believe you can use the same logic inside the map function.

Reply:
Because I need the indexes. I can technically use the `map` function by converting from list to a map, but this makes the code less efficient and less readable, from what I tested